cli/popper/commands/cmd_add.py

```diff
@@ -0,0 +1,74 @@
+import click
+import os
+import hcl
+import shutil
+import popper.utils as pu
+import popper.scm as scm
+from popper.cli import pass_context
+from distutils.dir_util import copy_tree
+
+
+@click.command('add', short_help='Import workflow from remote repo.')
+@click.argument('path', required=True)
+@pass_context
+def cli(ctx, path):
+    """Imports a workflow from a remote project to the current project
+    directory.
+    """
+    project_root = scm.get_root_folder()
+    if not pu.is_popperized(project_root):
+        pu.fail('Repository has not been popperized.')
+        return
+
+    parts = path.split('/')
+    parts = list(filter(None, parts))
+    if len(parts) < 3:
```

Comment:
similar to how `uses` is treated in a workflow, we can relax the requirement so that when the URL is omitted, it defaults to `github.com`. This is already handled by the `parse` function

Reply:
ok